<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_device_control</class>
 <widget class="QDialog" name="dialog_device_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>292</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid_control">
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_control_device_name">
      <property name="text">
       <string>Device:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_control_state_off">
      <property name="text">
       <string>Turn Off</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_control_state_on">
      <property name="text">
       <string>Turn On</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLabel" name="lbl_control_device_name_current">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="lbl_control_device_state_current">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="dglBtn_control">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
